tests/kernel/run_ltp.pm

```diff
@@ -260,7 +260,7 @@ sub run {
     assert_script_run('cd /opt/ltp/testcases/bin');
 
     if ($is_network) {
-        script_run('systemctl stop wicked wickedd NetworkManager');
+        script_run('systemctl stop NetworkManager SuSEfirewall2 wicked wickedd');
```

Comment:
it's good you ask. Sorry if my comments seem to not follow the same principles in different PRs. The "coding style" is mentioned in https://github.com/os-autoinst/os-autoinst-distri-opensuse#coding-style so there **is** a style guide. It also refers to https://github.com/os-autoinst/os-autoinst#how-to-contribute which references [How to Write a Git Commit Message](http://chris.beams.io/posts/git-commit/). This should answer what to put into commit messages but does not answer what should be in the code itself.

My proposal is like this:
* git commit message should describe the necessity or benefit of the *change* of code
* infile comments should describe the *current state* of code

Additionally: comments in pull requests consequently describe the *changes of the changes*, e.g. what you changed when you rewrote commits (which describe change).

Also for infile comments I would say:
1. Best is if the code describes itself in an obvious way
2. If 1. is not possible add comments